_rules/SC1-2-3+SC1-2-5-Video-description-track.md

```diff
@@ -50,7 +52,7 @@ This means that the rule can only provide a pass for these succescriteria if acc
 [audio output]: ../pages/algorithms/audio-output.html
 [visual output]: ../pages/algorithms/visual-output.html
 [non-streaming]: ../pages/algorithms/non-streaming-media-element.html
-[visible in the page]: ../pages/algorithms/visible-in-the-page.html
+[visible on the page]: ../pages/algorithms/visible-on-the-page.html
```

Comment:
https://github.com/auto-wcag/auto-wcag/pull/147
The way relative links to algorithms are referred, needs to change based on above PR.
Eg:
[non-empty](#non-empty)